src/index.html

```diff
@@ -11,7 +11,158 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+
+    <form action="https://localhost:3000/api" method="post">
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">Surname:</label>
+        <input
+          type="text"
+          name="surname"
+          id="surname"
+          autocomplete="off"
+          required
+        ><br>
```

Comment:
```suggestion
        >
```

no need `br`
please remove them